Streetcode/Streetcode.WebApi/appsettings.IntegrationTests.json

```diff
@@ -1,6 +1,6 @@
 {
   "ConnectionStrings": {
-  "DefaultConnection": "Server=DESKTOP-I7Q35NQ\\SQLEXPRESS;Database=StreetcodeDbtest;User Id=sa;Password=Admin@1234;MultipleActiveResultSets=true;TrustServerCertificate=true"
+    "DefaultConnection": "Server=localhost;Database=Streetcode_IntegrationTests_Db;User Id=sa;Password=sdlLKMCD234!@#lkcmds;MultipleActiveResultSets=True;TrustServerCertificate=true"
```

Comment:
Maybe, dont commit appsetings file with your credentials without any reasons
and if you change appsetings file, write warning in comment to PR

Reply:
Hi, new connection string( `Server=localhost;Database=Streetcode_IntegrationTests_Db` ) has connection data used for connecting to sql server in docker container for running integration tests in pr check,
P.S. old connection data is the same as in develop branch( i mean `Server=DESKTOP-I7Q35NQ\\SQLEXPRESS;Database=StreetcodeDbtest`)
P.S. new password( `sdlLKMCD234!@#lkcmds` ) is needed to connect to sql server in docker contaner when we run integration tests in pr check. New `User Id` and `Password` are defualt values for connecting to sql server in pr check.